network-api/networkapi/wagtailpages/pagemodels/base.py

```diff
@@ -468,6 +486,25 @@ class Homepage(FoundationMetadataPageMixin, Page):
         blank=True
     )
 
+    # Partner Section
+    partner_heading = models.CharField(max_length=75, default='Partner with us')
+    partner_intro_text = models.TextField(blank=True)
+    partner_link_text = models.CharField(max_length=35, default="Let's work together")
+    partner_internal_link = models.ForeignKey(
+        'wagtailcore.Page',
+        blank=True,
+        null=True,
+        on_delete=models.SET_NULL,
+        related_name='parnter_internal_link',
+    )
+    partner_external_link = models.URLField(blank=True)
```

Comment:
Is this for the "Let's work together" link? If so, as I understand it that will always be an internal link, to explain our side of that story (which would obviate a lot of the code for dealing with the different types of links =D).

@taisdesouzalessa is that correct?

Reply:
Based on https://github.com/mozilla/foundation.mozilla.org/issues/4844#issuecomment-662038941, we'll only need the ForeignKey field with the PageChooser, without the additional code for dual-wielding external links as well =)